examples/puppet_headnode/manifests/static/boot_notify.py

```diff
@@ -2,8 +2,8 @@
 from flask import Flask
 app = Flask(__name__)
 
-import re
-import os
+import re  # noqa
+import os  # noqa
```

Comment:
Ok. In this case I don't think there's any reason for them not to be; let's just move them.